Docs: add troubleshooting for mysql (#11012)
airbytehq/airbyte: docs/integrations/destinations/mysql.md

* To sync data to MySQL **with** normalization MySQL database 8.0.0 or above
* To sync data to MySQL **without** normalization you'll need MySQL 5.0 or above.

#### Troubleshooting

> Some users reported they could not connect to Amazon AWS RDS MySQL or MariaDB. When creating the new connector they receive a `Cannot create a PoolableConnectionFactory`.
> To solve this issue add `enabledTLSProtocols=TLSv1.2` in the JDBC parameters.

Reviewer: ```suggestion
Some users reported that they could not connect to Amazon RDS MySQL or MariaDB. This can be diagnosed with the error message: `Cannot create a PoolableConnectionFactory`.
To solve this issue add `enabledTLSProtocols=TLSv1.2` in the JDBC parameters.
```
